packages/components/input-number/src/input-number.vue

```diff
@@ -275,7 +275,7 @@ watch(
   (value) => {
     const userInput = verifyValue(data.userInput)
     const newValue = verifyValue(value, true)
-    if (!userInput || userInput !== newValue) {
+    if (userInput !== 0 && (!userInput || userInput !== newValue)) {
```

Comment:
@StephenKe maybe we can use `isNumber(userInput)` here for this branch. Something like:

```ts
if (!isNumber(userInput) && !userInput || otherConditions) {}
```

Reply:
Plus, since this is a good catch, I'd like to ask you to write a test case for this edge case.

Reply:
> Plus, since this is a good catch, I'd like to ask you to write a test case for this edge case.

Added a edge case in change event.But wrapper.find('input').setValue(0) cannot catch it.I wonder if it should test props.modelValue in watch.